vllm/v1/core/sched/scheduler.py

```diff
@@ -38,6 +39,39 @@
 logger = init_logger(__name__)
 
 
+def remove_all(lst: list, items_to_remove: set) -> list:
```

Comment:
Could you move this method to https://github.com/vllm-project/vllm/blob/main/vllm/v1/core/sched/utils.py?